### PR TITLE
Creating police section for major Canadian police forces

### PR DESCRIPTION
**Pull request by bkissin**
The RCMP (Royal Canadian Mounted Police) is the police force in most of Canada, except for Ontario and Quebec, which both have their own provincial police forces. Edits were made in the online client, so apologies in advance for possible syntax errors. Getting back on this project after a while away.

### CODE REVIEW

**Comment by bhousel** [comment]
This is a good idea, but I'm not quite ready for this.. 

Would police be more of a `brand` or `operator` ?  We're currently trying to stick to brands, but I want to expand to operators (to better handle things like public transport and hospital networks - police are probably in here too).  I know the distinction in OSM is really thin.

**Comment by doublah** [comment]
`operator` seems more appropriate for Police and other non-retail/business tags in my opinion

**Comment by bkissin** [comment]
No worries! If we want to hold off on making this change until ready, I am fine with that.

File: brands/amenity/police [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "amenity/police|RCMP": {
+    "countryCodes": ["ca"],
+    "tags": {
+      "amenity": "police",
+      "brand": "Royal Canadian Mounted Police",
+      "brand:wikidata": "Q335175",
+      "brand:wikipedia": "en:Royal Canadian Mounted Police",
+      "official_name": "Royal Canadian Mounted Police",
```
Comment (anandfresh) [inline brands/amenity/police]: Shall we stick to camelCase for property names ?

Comment (bhousel) [reply brands/amenity/police]: Yes

File: brands/amenity/police [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "amenity/police|RCMP": {
+    "countryCodes": ["ca"],
+    "tags": {
+      "amenity": "police",
+      "brand": "Royal Canadian Mounted Police",
```
Comment (matkoniecz) [inline brands/amenity/police:6]: Is it really a brand? I would consider switch to `operator`

Comment (bhousel) [reply brands/amenity/police:6]: well yes, that is the whole reason this PR has sat since November

Comment (matkoniecz) [reply brands/amenity/police:6]: I know, I wanted to ping author.

**Comment by bhousel** [comment]
Going to close this for now.  

Restructuring the code to allow `operator` or `network` to be the the defining tag (instead of `brand`) is in the back of my mind and will happen eventually.

**Pull request closed without merging**